cmd/kdk/init.go

```diff
@@ -34,8 +34,9 @@ var initCmd = &cobra.Command{
 }
 
 func init() {
+	CurrentKdkEnvConfig.ConfigFile.AppConfig.Port = kdk.Port
+
 	initCmd.Flags().StringVarP(&CurrentKdkEnvConfig.ConfigFile.AppConfig.Name, "name", "n", "kdk", "KDK Name")
-	initCmd.Flags().StringVarP(&CurrentKdkEnvConfig.ConfigFile.AppConfig.Port, "port", "p", "2022", "KDK Port")
```

Comment:
Shouldn't it still be possible to override with flag?  Default could be read back from conf, yeah?